src/store/modules/events/index.ts

```diff
@@ -450,6 +473,23 @@ const module: Module<EventsModuleState, RootState> = {
       state.repetitions[key] = repetition;
     },
 
+    /**
+     * Mark event as visited in state
+     *
+     * @param {EventsModuleState} state
+     * @param {string} projectId - project event is related to
+     * @param {string} eventId - visited event
+     * @param {string} userId - user who visited event
+     */
+    [MutationTypes.MARK_AS_VISITED](state, { projectId, eventId, userId }) {
+      const key = projectId + ':' + eventId;
```

Comment:
It is not a good practice that forming a key of the `list` object is not in a single place.